deploy/kubei.yaml

```diff
@@ -40,115 +40,107 @@ roleRef:
-      containers:
-      - name: clair
-        image: gcr.io/eticloud/k8sec/clair-scan-server:2.1.6
-        imagePullPolicy: Always
-        ports:
-        - containerPort: 6060
-        - containerPort: 6061
-        resources:
-          limits:
-            cpu: 2000m
-            memory: 6G
-          requests:
-            cpu: 700m
-            memory: 3G
-# uncomment the following lines in case Kubei is running behind proxy. update PROXY_IP & PROXY_PORT according to your proxy settings
-#        env:
-#        - name: HTTPS_PROXY
-#          value: "{ PROXY_IP }:{ PROXY_PORT }"
-#        - name: HTTP_PROXY
-#          value: "{ PROXY_IP }:{ PROXY_PORT }"
-#        - name: NO_PROXY
-#          value: "postgres"
----
-apiVersion: v1
-kind: Service
 metadata:
   labels:
-    app: postgres
-  name: postgres
+    app: grype-server
+  name: grype-server
   namespace: kubei
 spec:
   type: ClusterIP
   ports:
-    - name: db
-      port: 5432
+    - name: grype-server
+      port: 9991
+      protocol: TCP
+    - name: health
+      port: 8080
       protocol: TCP
+      targetPort: 8080
   selector:
-    app: postgres
+    app: grype-server
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: clair-postgres
+  name: grype-server
   namespace: kubei
   labels:
-    app: postgres
+    app: grype-server
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: postgres
+      app: grype-server
   template:
     metadata:
       labels:
-        app: postgres
-        kubeiShouldScan: "false"
+        app: grype-server
+        kubeiShouldScan: 'false'
     spec:
+      initContainers:
+        - name: init-grype-server
+          image: yauritux/busybox-curl
+          securityContext:
+            capabilities:
+              drop:
+                - ALL
+            runAsNonRoot: true
+            runAsGroup: 1001
+            runAsUser: 1001
+            privileged: false
+            allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: true
+          args:
+            - /bin/sh
+            - '-c'
+            - >
+              set -x; while [ $(curl -sw '%{http_code}'
+              "http://grype-server.kubei:8080/healthz/ready" -o /dev/null) -ne
+              200 ]; do
+                echo "waiting for grype-server to be ready";
+                sleep 15;
+              done
       containers:
-      - name: clair-db  
-        image: gcr.io/eticloud/k8sec/clair-db:12.2.ubi.6
-        imagePullPolicy: Always
-        ports:
-        - containerPort: 5432
-        resources:
+        - name: grype-server
+          image: 'gcr.io/eticloud/k8sec/grype-server:2.1.6'
+          args:
+            - run
+            - '--log-level'
+            - warning
+          env:
+            - name: DB_ROOT_DIR
+              value: /tmp/
+            - name: HTTPS_PROXY
+              value: '{ PROXY_IP }:{ PROXY_PORT }'
+            - name: HTTP_PROXY
+              value: '{ PROXY_IP }:{ PROXY_PORT }'
```

Comment:
forgot to comment out it and missed the `# uncomment the following lines in case Kubei is running behind proxy. update PROXY_IP & PROXY_PORT according to your proxy settings`